SCClassLibrary/Common/Control/ServerStatus.sc

```diff
@@ -5,7 +5,7 @@ ServerStatusWatcher {
 	var <>notified = false, <notify = true;
 	var alive = false, <aliveThread, <>aliveThreadPeriod = 0.7, statusWatcher;
 
-	var <serverRunning = false, <>serverBooting = false, <unresponsive = false;
```

Comment:
again, are we ok to delete/rename this public instance member?

Reply:
The getter is still there, so nothing in the code interface is removed. 
it is now a method in line 204:
```
- 	serverRunning_ { | running |
+	serverRunning { ^hasBooted and: notified }
+ 	serverRunning_ { | running |
```
The PR raises the demands for serverRunning, it now requires that notified be true.

Reply:
got it, thanks!